src/resolution.js

```diff
@@ -25,6 +32,8 @@ var getScreenTypeObject = function (type) {
 * @module enyo/resolution
 */
 var ri = module.exports = {
+	config: {},
```

Comment:
Fair enough - let's leave this in case the config init code changes or is removed, later.